src/thunderbots/software/CMakeLists.txt

```diff
@@ -41,6 +41,12 @@ file(GLOB SHARED_UTIL_SRC LIST_DIRECTORIES false CONFIGURE_DEPENDS
         ${CMAKE_CURRENT_SOURCE_DIR}/../shared/*.h
         )
 
+# Find the g3log library, the logger we use for our AI
+# See: https://github.com/KjellKod/g3log
+SET(G3_LIBRARY_PATH "/usr/local/include")
+message("g3log lib path: ${G3_LIBRARY_PATH}")
+find_library(G3LOG g3logger PATHS ${G3_LIBRARY_PATH})
```

Comment:
Would be nice if we fail early if the library wasn't found.

Reply:
I believe this will fail on this line if the library is not found.

Reply:
I thought it would just set `${G3_LIBRARY_PATH}` to be empty? https://cmake.org/cmake/help/v3.4/command/find_library.html

Reply:
Oh sorry, you're completely correct (just tested this out on my local machine to confirm). I was thinking of `find_package`.

Reply:
fixed